modules/apps/29-fee/client/cli/cli.go

```diff
@@ -18,6 +18,8 @@ func GetQueryCmd() *cobra.Command {
 		GetCmdTotalRecvFees(),
 		GetCmdTotalAckFees(),
 		GetCmdTotalTimeoutFees(),
+		GetIncentivizedPacketByPacketId(),
+		GetAllIncentivizedPackets(),
```

Comment:
I think we should use `GetCmd...` as the others

```suggestion
		GetCmdIncentivizedPacket(),
		GetCmdIncentivizedPackets(),
```